pinc/DPDatabase.inc

```diff
@@ -28,6 +28,9 @@ final class DPDatabase
         $sql = "SET SESSION sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));";
         if(!mysqli_query(self::$_connection, $sql))
             throw new DBConnectionError("Unable to set sql_mode");
+
+        // Autocommit DB updates
+        mysqli_autocommit(self::$_connection, TRUE);
```

Comment:
[The MySQL docs](https://dev.mysql.com/doc/refman/5.6/en/innodb-autocommit-commit-rollback.html) say:
> By default, MySQL starts the session for each new connection with autocommit enabled

So it seems like this statement is unnecessary, but I guess it can't hurt.

Also, this ignores the function's return value.Which is maybe okay, if we're not expecting the call to do anything?

Reply:
Yup, thanks - I missed that. I also confirmed that autocommit is being set to 1 already, so this change isn't needed.